cli/lsp/diagnostics.rs

```diff
@@ -1615,35 +1620,6 @@ let c: number = "a";
       .diagnostics
   }
 
-  #[tokio::test]
-  async fn test_cancelled_ts_diagnostics_request() {
-    let temp_dir = TempDir::new();
-    let (snapshot, cache_location) = setup(
-      &temp_dir,
```

Comment:
I don't understand why this test need to be removed...

Reply:
It no longer returns empty diagnostics but gives a cancelled error. Since it's not a meaningful or unique case anymore I just removed the test